atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/Cell.java

```diff
@@ -59,8 +59,8 @@ public static boolean isNameValid(byte[] name) {
     }
 
     private void validateNameValid(byte[] name) {
-        Preconditions.checkNotNull(name, "name cannot be null");
-        Preconditions.checkArgument(name.length > 0, "name must be non-empty");
+        com.palantir.logsafe.Preconditions.checkNotNull(name, "name cannot be null");
+        com.palantir.logsafe.Preconditions.checkArgument(name.length > 0, "name must be non-empty");
```

Comment:
why do we still require both `Preconditions` in this file? Unclear to me why the invocation on line 66 can't be converted also?